Modules/posixmodule.c

```diff
@@ -105,6 +105,11 @@ corresponding Unix manual entries for more information on calls.");
 #include <sched.h>
 #endif
 
+#ifdef HAVE_COPY_FILE_RANGE
+#include <sys/syscall.h>
```

Comment:
It seems that this version no longer uses `syscall`.

Reply:
Fixed in adf56cd